src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricSerializer.cs

```diff
@@ -25,18 +25,18 @@ namespace Dynatrace.OpenTelemetry.Exporter.Metrics
 {
     public class DynatraceMetricSerializer
     {
-        private readonly string prefix;
-        private readonly IEnumerable<KeyValuePair<string, string>> tags;
+        private readonly string _prefix;
+        private readonly IEnumerable<KeyValuePair<string, string>> _dimensions;
 
-        private const int MAX_LENGTH_METRIC_KEY = 250;
-        private const int MAX_LENGTH_DIMENSION_KEY = 100;
-        private const int MAX_LENGTH_DIMENSION_VALUE = 250;
-        private const int MAX_DIMENSIONS = 50;
+        private const int MaxLengthMetricKey = 250;
```

Comment:
Why change these from uppercase? Is this some `C#` convention? I personally prefer the UPPER_CASE style for constants.